### PR TITLE
bugfix: Fix the NPE when focus on the output channel

### LINKED ISSUE #337: NPE when resolving path in the tree view
This error happens when the user tries to focus on the output channel:

```
[Error - 10:36:38 AM] Sep 27, 2020, 10:36:38 AM Problems occurred when invoking code from plug-in: "org.eclipse.jdt.ls.core".
null
java.lang.NullPointerException
	at com.microsoft.jdtls.ext.core.model.PackageNode.createNodeForProject(PackageNode.java:152)
	at com.microsoft.jdtls.ext.core.PackageCommand.resolvePath(PackageCommand.java:206)
	at com.microsoft.jdtls.ext.core.CommandHandler.executeCommand(CommandHandler.java:33)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler$1.run(WorkspaceExecuteCommandHandler.java:215)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler.executeCommand(WorkspaceExecuteCommandHandler.java:205)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$3(JDTLanguageServer.java:495)
	at org.eclipse.jdt.ls.core.internal.BaseJDTLanguageServer.lambda$0(BaseJDTLanguageServer.java:75)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

### PR DESCRIPTION
**Pull request by jdneo**
fix #337

### CODE REVIEW

File: src/views/dependencyExplorer.ts [hunk 1]
```diff
@@ -29,7 +29,10 @@ export class DependencyExplorer implements Disposable {
         context.subscriptions.push(
             window.onDidChangeActiveTextEditor((textEditor: TextEditor) => {
                 if (this._dependencyViewer.visible && textEditor && textEditor.document && Settings.syncWithFolderExplorer()) {
-                    this.reveal(textEditor.document.uri);
+                    const uri: Uri = textEditor.document.uri;
+                    if (uri.scheme === "file") {
```
Comment (testforstephen) [inline src/views/dependencyExplorer.ts:33]: add `jdt` to cover the content provider for .class files?

Comment (jdneo) [reply src/views/dependencyExplorer.ts:33]: make sense

Comment (testforstephen) [reply src/views/dependencyExplorer.ts]: Can the `textEditor.document.uri` be empty? If opening an Untitled file, what's the uri like?

Comment (jdneo) [reply src/views/dependencyExplorer.ts]: It's `'untitled:Untitled-1'`

**Review by testforstephen — APPROVED** [review]

**Pull request merged**